test/acceptance/tests/basic/basic_test.go

```diff
@@ -751,30 +752,55 @@ func TestValidation_MeshGateway(t *testing.T) {
 func TestBasic(t *testing.T) {
 	t.Parallel()
 
-	cfg := suite.Config()
-
 	cases := []struct {
 		secure        bool
+		enterprise    bool
 		stateFile     string
 		ecsClusterARN string
 		datacenter    string
 	}{
-		// Works for 2 clusters in parallel. Each test case should have a unique cluster
-		// and requires a unique Terrform state file. The datacenter must be unique within
-		// the VPC to avoid conflicts in CloudMap namespaces.
-		{false, "terraform.tfstate", cfg.ECSClusterARN, "dc1"},
-		{true, "terraform-secure.tfstate", cfg.ECSCluster2ARN, "dc2"},
+		{secure: false},
+		{secure: true},
+		{secure: true, enterprise: true},
 	}
-	for _, c := range cases {
+
+	cfg := suite.Config()
+	require.GreaterOrEqual(t, len(cfg.ECSClusterARNs), len(cases),
+		"TestBasic requires %d ECS clusters. Update setup-terraform and re-run.", len(cases),
+	)
+
+	initOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "./terraform/basic-install",
```

Comment:
I'll probably actually want to add separate Terraform config for other enterprise tests.

I'd like to have an enterprise test with the dev-server that replicates the admin partition test that we have for HCP - so two clusters/partitions with a service in each.